src/mlpack/bindings/utils/camel_case.hpp

```diff
@@ -1,25 +1,24 @@
 /**
- * @file bindings/go/camel_case.hpp
- * @author Yashwant Singh
+ * @file bindings/utils/camel_case.hpp
+ * @author Yashwant Singh Parihar
  *
- * Given a C++ typename that may have template parameters, return stripped and
- * printable versions to be used in Go bindings.
+ * Convert snake_case name to CamelCase.
  *
  * mlpack is free software; you may redistribute it and/or modify it under the
  * terms of the 3-clause BSD license.  You should have received a copy of the
  * 3-clause BSD license along with mlpack.  If not, see
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
-#ifndef MLPACK_BINDINGS_GO_CAMEL_CASE_HPP
-#define MLPACK_BINDINGS_GO_CAMEL_CASE_HPP
+#ifndef MLPACK_BINDINGS_UTILS_CAMEL_CASE_HPP
+#define MLPACK_BINDINGS_UTILS_CAMEL_CASE_HPP
```

Comment:
This is a nice cleanup---I appreciate that you've done this.  It reduces the overall amount of code which is great.  :+1:

Reply:
Thank You.